.github/workflows/pr_check_client_side_changes.yml

```diff
@@ -30,7 +30,7 @@ jobs:
             const title = '[Ports]: Backport client side changes';
             for (const repo of ['playwright-python', 'playwright-java', 'playwright-dotnet']) {
               const { data: issuesData } = await github.rest.search.issuesAndPullRequests({
-                q: `is:issue is:open repo:microsoft/${repo} in:title "${title}"`
+                q: `is:issue is:open repo:microsoft/${repo} in:title "${title}" author:playwrightmachine"`
```

Comment:
I regularly end up creating such issue manually, to carry over some of the accumulated changes. Does it mean those issues will not work? I'd rather we fixed that.